static/app/components/performanceOnboarding/sidebar.tsx

```diff
@@ -45,6 +58,12 @@ function PerformanceOnboardingSidebar(props: CommonSidebarProps) {
   const {projectsWithoutFirstTransactionEvent, projectsForOnboarding} =
     filterProjects(projects);
 
+  const priorityProjectIds: Set<string> | null = useMemo(() => {
+    const queryParams = qs.parse(location.search);
+    const decodedProjectIds = decodeProjectIds(queryParams.project);
```

Comment:
I think the `useLocation` hook is preferred here if possible rather then the global location object. In general id use react api's over browser apis.

you can add `const useLocation = location<{project: string | string[]}>`

Reply:
Yeah I can switch to `useLocation` here